cmd/tailscaled/tailscaled.go

```diff
@@ -403,14 +403,16 @@ func run() (err error) {
 		debugMux = newDebugMux()
 	}
 
-	sys.Set(tailfsimpl.NewFileSystemForRemote(logf))
-
 	return startIPNServer(context.Background(), logf, pol.PublicID, sys)
 }
 
 var sigPipe os.Signal // set by sigpipe.go
 
 func startIPNServer(ctx context.Context, logf logger.Logf, logID logid.PublicID, sys *tsd.System) error {
+	// Set up tailfs.FileSystemForRemote here so that we get it on both Windows
+	// and UNIX.
+	sys.Set(tailfsimpl.NewFileSystemForRemote(logf))
```

Comment:
We also get it on iOS. Are we going to be a file server there too?

Reply:
No, we don't want to be a file server there. Let me fix.

Reply:
Actually, I don't see us importing `cmd/tailscaled` on iOS. What makes you say we'll get this on IOS?